SWLOR.Game.Server/Event/Conversation/HasCredits.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using NWN;
+using SWLOR.Game.Server.Data.Contracts;
+using SWLOR.Game.Server.GameObject;
+using SWLOR.Game.Server.Service.Contracts;
+using Object = NWN.Object;
+
+namespace SWLOR.Game.Server.Event.Conversation
+{
+    public class HasCredits : IRegisteredEvent
+    {
+        private readonly INWScript _;
+
+        public HasCredits(INWScript script)
+        {
+            _ = script;
+        }
+
+        public bool Run(params object[] args)
+        {
+                NWPlayer oPC = _.GetPCSpeaker();
+                NWObject oNPC = Object.OBJECT_SELF;
+
+
+
+                int nGold = _.GetGold(oPC);
+                int reqGold = _.GetLocalInt(oNPC,"gold");
+                if (nGold > reqGold)
+                {
+                return true;
```

Comment:
Probably related to the extra whitespace, but make sure you tab the return true so it's not indented on the same line as the bracket.s